example/code1/example.go

```diff
@@ -18,9 +18,9 @@ func main() {
 		ContentTypeNosniff:    true,
 		BrowserXssFilter:      true,
 		ContentSecurityPolicy: "default-src 'self'",
-		ReferrerPolicy:        "strict-origin-when-cross-origin",
```

Comment:
Why remove this code?

Reply:
It's a duplicate field declaration. You're declaring `ReferrerPolicy` twice.

Reply:
Becausee it's a duplicate field declaration, you're declaring `ReferrerPolicy` twice.